acceptance/indy_acceptance/indy-tests/utils/constant.py

```diff
@@ -0,0 +1,44 @@
+'''
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+'''
+
+
+class Colors:
+    """ 
+    Class to set the colors for text.
+    Syntax:  print(Colors.OKGREEN +"TEXT HERE" +Colors.ENDC)
+    """
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'  # Normal default color
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+class Roles:
+    """ 
+    Class to define roles.
+    """
+    TRUSTEE = "TRUSTEE"
+    STEWARD = "STEWARD"
+    TRUST_ANCHOR = "TRUST_ANCHOR"
+    TGB = "TGB"  # obsolete
+    NONE = ""
+
+
+class Constant:
+    """ 
+    Class Constant store some necessary paths.
+    """
+    import os
+    work_dir = os.path.expanduser('~') + os.sep + ".indy"
```

Comment:
The current working dir for libindy is .indy_client, not .indy (in the latest stable)

Reply:
We only see the ".indy" directory. We are using lib-indy version 1.0.1 . This variable is defined in file "constant.py", users can change if needed.